blues/smartdart.py

```diff
@@ -193,16 +524,16 @@ def reDart(self, changevec):
         """
         Helper function to choose a random dart and determine the vector
         that would translate the COM to that dart center
-        """ 
+        """
         dartindex = np.random.randint(len(self.dartboard))
         dvector = self.dartboard[dartindex]
-        chboard = dvector + changevec   
+        chboard = dvector + changevec
         print('chboard', chboard)
         return chboard
 
     def dartmove(self, context=None, residueList=None):
         """
-        Obsolete function kept for reference. 
+        Obsolete function kept for reference.
```

Comment:
@sgill2 - why are we keeping an obsolete function? If you need it in the future, it will always be there in the history on github (e.g. you can recover it from an old commit).